common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -400,8 +428,12 @@ private static async Task RunTestsAsync(IPerfTest[] tests, PerfOptions options,
             var secondsPerOperation = 1 / operationsPerSecond;
             var weightedAverageSeconds = totalOperations / operationsPerSecond;
 
+            var cpuElapsed = cpuStopwatch.Elapsed.TotalMilliseconds;
+            var cpuTime = (Process.GetCurrentProcess().TotalProcessorTime - startCpuTime).TotalMilliseconds;
+            var cpuPercentage = (cpuTime / cpuElapsed) / Environment.ProcessorCount;
```

Comment:
Since this is `cpuPercentage`, the value should be multiplied by 100 here.

Reply:
The value is multiplied by 100 when printing.  I prefer to keep the variable as a double between 0 and 1.

Reply:
I saw that. My comment is that since the name is `cpuPercentage` it should be so. If you prefer to multiply during logging, maybe the variable should be called `cpuRatio` or something. Bit weird to see a percentage variable being multiplied by 100.

Reply:
I understand your point, but I think the most common convention is for percentages to be stored numerically as a value where `1.0 == 100%`, and then multiplied by 100 when formatting as a percentage string.  This is how the .NET [percent format specifier](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#percent-format-specifier-p) works (and also Excel).

This code previously was using the .NET percent format specifier, however the formatting varies by culture (`1.23%` vs `1.23 %`).  To ensure columns align regardless of culture, I decided to use the number format specifier instead.